src/combat.ts

```diff
@@ -289,6 +289,12 @@ export class Macro extends StrictMacro {
           have($skill`Gallapagosian Mating Call`),
         Macro.if_($monster`garbage tourist`, Macro.trySkill($skill`Gallapagosian Mating Call`))
       )
+      .externalIf(
+        get("longConMonster") !== $monster`garbage tourist` &&
```

Comment:
can we get the log from the dude who ran this in AWOL to make sure it used long con five times? i could see this maybe not actually working if mafia's settings don't properly clear longConMonster when the effect wears off.

Reply:
Does Long Con wear off, theres nothing on the wiki page about it being limited?